eml/api/api.js

```diff
@@ -127,6 +127,23 @@ export const getExercisesInSection = async (courseId, sectionId) => {
   }
 };
 
+//CREATED BY VIDEOSTREAM TEAM
```

Comment:
There is no reason to fix it now, however, at some point I think we can remove these comments since it would be annoying if every group had to comment on what functions they create.